webpack/assets/javascripts/react_app/components/HostsIndex/Columns/reportedDataColumns.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable camelcase */
+import { number_to_human_size as NumberToHumanSize } from 'number_helpers';
+import { translate as __ } from '../../../common/I18n';
+
+const reportedDataColumns = [
+  {
+    columnName: 'model',
+    title: __('Model'),
+    wrapper: hostDetails =>
+      hostDetails?.compute_resource_name || hostDetails?.model_name,
+    isSorted: true,
+    weight: 1000,
+  },
+  {
+    columnName: 'sockets',
+    title: __('Sockets'),
+    wrapper: hostDetails => hostDetails?.reported_data?.sockets,
+    isSorted: false,
+    weight: 1100,
+  },
+  {
+    columnName: 'cores',
+    title: __('Cores'),
+    wrapper: hostDetails => hostDetails?.reported_data?.cores,
+    isSorted: false,
+    weight: 1200,
+  },
+  {
+    columnName: 'ram',
+    title: __('RAM'),
+    wrapper: hostDetails => {
+      if (!hostDetails?.reported_data?.ram) return null;
+      return NumberToHumanSize(hostDetails.reported_data.ram * 1024 * 1024, {
+        strip_insignificant_zeros: true,
+      });
+    },
+    isSorted: false,
+    weight: 1300,
+  },
+  {
+    columnName: 'virtual',
+    title: __('Virtual'),
+    wrapper: hostDetails => {
+      const value = hostDetails?.reported_data?.virtual;
+      if (value === undefined || value === null) {
+        return '';
+      }
+      if (value) {
+        return __('Virtual');
+      }
+      return __('Physical');
```

Comment:
Isn't using a ternary a lot shorter and clearer?
```suggestion
      return value ? __('Virtual') :__('Physical');
```

Reply:
It was refactored blindly. Did not review the code.